package.json

```diff
@@ -14,6 +14,8 @@
     "stylelint-config-standard": "^20.0.0"
   },
   "dependencies": {
+    "@expo/vector-icons": "^10.2.0",
```

Comment:
why in global package.json

Reply:
why not

Reply:
because it's not needed there. it's needed only for your app